src/components/shared/utilities/GetTranslate.tsx

```diff
@@ -0,0 +1,9 @@
+import { FormattedMessage } from 'react-intl'
+
+const GetTranslate = ({ message }: { message: string }) => {
```

Comment:
I don't see the benefit of a component whose only purpose is to render a `FormattedMessage`. Why not just do that directly?